factory-containers/build.sh

```diff
@@ -68,7 +68,7 @@ for x in $IMAGES ; do
 			status "Detected changes to $x"
 		else
 			status "No changes to $x, tagging only"
-			no_op_tag=1
+			# no_op_tag=1
```

Comment:
Is this a typo?

Reply:
Heh, I was just going to ask the same question

Reply:
great catch. I commented that out to force some local debugging to work.